docs/docs/spec.md

```diff
@@ -146,16 +141,18 @@ A nullness operator is one of 4 values:
 >     -   One way to conceptualize this is that `String NO_CHANGE` means
 >         "non-null `String`" but that `T NO_CHANGE` means "nullness comes from
 >         the value of `T`."
-> -   `UNSPECIFIED`: This is the operator produced by not putting `@Nullable` on
->     a type usage *in code that is outside a [null-marked scope]*. Roughly, it
->     is the operator assigned to "completely unannotated code."
+> -   `UNSPECIFIED`: This is the operator produced for a type usage *outside* of
```

Comment:
I'm kind of so-so on this change. There was some parallelism to `NO_CHANGE`, though I admit it wasn't great. If not for the "FIX THAT" part, I'd probably say to go for it, but maybe just revert so that we can land the rest of the PR?

Reply:
sounds amazing, will do